Packs/MicrosoftExchangeOnline/Integrations/EwsExtensionEXOPowershellV3/EwsExtensionEXOPowershellV3.yml

```diff
@@ -2337,6 +2434,81 @@ script:
       description: The value of the entry that was removed.
       type: string
     description: Remove items from the Tenant Allow/Block Lists.   You can delete items by their value or by unique ID.  Uses PowerShell cmdlet Remove-TenantAllowBlockListItems cmdlet.
```

Comment:
Remove items from the tenant's Allow/Block Lists. You can delete items by their value or by their unique ID.  Uses the PowerShell cmdlet Remove-TenantAllowBlockListItems cmdlet